### PR TITLE
Pipeline common step definitions

### PR DESCRIPTION
**Pull request by sawantpritam**
@itsankit-google @rmstar  Please review.

### CODE REVIEW

**Review by itsankit-google — APPROVED** [review]

File: src/main/java/stepsdesign/PipelineSteps.java [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+      BeforeActions.scenario.write(CdfPipelineRunAction.captureRawLogs());
+      PrintWriter out = new PrintWriter(BeforeActions.file);
+      out.println(CdfPipelineRunAction.captureRawLogs());
+    } catch (Exception e) {
+      BeforeActions.scenario.write("Exception in capturing logs : " + e);
+    }
+  }
+
+  @Then("Validate successMessage is displayed")
+  public void validateSuccessMessageIsDisplayed() {
```
Comment (itsankit-google) [inline src/main/java/stepsdesign/PipelineSteps.java]: Please remove this step, don't need this redundant step. `Verify the pipeline status is succeeded` is enough.

**Review by itsankit-google — COMMENTED** [review]
LGTM, one minor comment.

File: src/main/java/stepsdesign/PipelineSteps.java (hunk 1 above)
Comment (sawantpritam) [reply src/main/java/stepsdesign/PipelineSteps.java]: removed

Comment (itsankit-google) [reply src/main/java/stepsdesign/PipelineSteps.java]: Please squash the commits

Comment (sawantpritam) [reply src/main/java/stepsdesign/PipelineSteps.java]: Done.

**Pull request merged**